mac

```diff
@@ -64,6 +64,16 @@ install_gui_app() {
   fi
 }
 
+# Check for Terminal running in ARM mode, install Rosetta, exit script
+ARCH=$(arch)
+if [[ "$ARCH" == "arm64" ]]; then
+    /usr/sbin/softwareupdate --install-rosetta --agree-to-license
+    fancy_echo "\e[1;34m *** CodeClan - \e[1;31mAdditional manual steps required please visit - \033[0m"
```

Comment:
So will this trigger the install and then display the message to look at the apple silicon markdown?

Reply:
Yeah basically.